src/include/sof/lib/alloc.h

```diff
@@ -44,13 +44,21 @@
  *
  * 4) Buffer Zone. Largest heap zone intended for audio buffers.
  *
+ * 5) Runtime Shared Zone. Similar to Runtime Zone, but content may be used and
+ * fred from any enabled core.
+ *
+ * 6) Runtime System Zone. Similar to System Zone, but content may be used from
+ * any enabled core.
```

Comment:
I feel like the comment for the runtime system shared zone isn't exactly synced up with the code.

Reply:
Why?

Reply:
Missing the "shared" keyword?